src/skin/options.html

```diff
@@ -150,6 +150,12 @@ <h1><span class="i18n_options_title"></span></h1>
           <span class="i18n_options_social_widgets_checkbox"></span>
         </label>
       </div>
+      <div class="checkbox">
+        <label>
+          <input type="checkbox" id="enable_dnt_checkbox">
+          <span class="i18n_options_enable_dnt_checkbox"></span>
+        </label>
+      </div>
       <div class="checkbox">
```

Comment:
Could you indicate that checking DNT policies is a sub-setting of sending the DNT signal? Perhaps by indenting the DNT policies checkbox a bit.

Reply:
Yes, indented this option in, flush with the footer text.